datahub-web-react/src/app/ingest/source/builder/RecipeForm/unity_catalog.tsx

```diff
@@ -138,7 +138,7 @@ export const UNITY_TABLE_ALLOW: RecipeField = {
 
 const tableDenyFieldPath = 'source.config.metastore_id_pattern.deny';
 export const UNITY_TABLE_DENY: RecipeField = {
-    name: 'catalog_pattern.allow',
+    name: 'table_pattern.allow',
```

Comment:
Should this be `table_pattern.deny`? Also, do we need to change the field path?. Same goes for `tableAllowFieldPath`

Reply:
updated